crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -347,6 +347,91 @@ module deepbook::clob_v2 {
         )
     }
 
+    fun create_pool_with_return_<BaseAsset, QuoteAsset>(
+        taker_fee_rate: u64,
+        maker_rebate_rate: u64,
+        tick_size: u64,
+        lot_size: u64,
+        creation_fee: Balance<SUI>,
+        ctx: &mut TxContext,
+    ) : Pool<BaseAsset, QuoteAsset> {
+        let base_type_name = type_name::get<BaseAsset>();
+        let quote_type_name = type_name::get<QuoteAsset>();
+
+        assert!(clob_math::unsafe_mul(lot_size, tick_size) > 0, EInvalidTickSizeLotSize);
+        assert!(base_type_name != quote_type_name, EInvalidPair);
+        assert!(taker_fee_rate >= maker_rebate_rate, EInvalidFeeRateRebateRate);
+
+        let pool_uid = object::new(ctx);
+        let pool_id = *object::uid_as_inner(&pool_uid);
+    
+        event::emit(PoolCreated {
+            pool_id,
+            base_asset: base_type_name,
+            quote_asset: quote_type_name,
+            taker_fee_rate,
+            maker_rebate_rate,
+            tick_size,
+            lot_size,
+        });
+        Pool<BaseAsset, QuoteAsset> {
+            id: pool_uid,
+            bids: critbit::new(ctx),
+            asks: critbit::new(ctx),
+            next_bid_order_id: MIN_BID_ORDER_ID,
+            next_ask_order_id: MIN_ASK_ORDER_ID,
+            usr_open_orders: table::new(ctx),
+            taker_fee_rate,
+            maker_rebate_rate,
+            tick_size,
+            lot_size,
+            base_custodian: custodian::new<BaseAsset>(ctx),
+            quote_custodian: custodian::new<QuoteAsset>(ctx),
+            creation_fee,
+            base_asset_trading_fees: balance::zero(),
+            quote_asset_trading_fees: balance::zero(),
+        }
+    }
+
+    public fun create_pool_with_return<BaseAsset, QuoteAsset>(
```

Comment:
Maybe `create_external_pool` + comment explaining who should use this (someone who wants to create a wrapped pool for use in an external protocol)?